src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -163,8 +163,21 @@ public async Task<Result<int>> Handle(CreateInvitationCommand request, Cancellat
                         participant.SortKey));
                     if (fr.UsePersonalEmail != null && fr.UsePersonalEmail == false && fr.Email != null)
                     {
-                        participants.Add(new BuilderParticipant(ParticipantType.Required,
-                            new ParticipantIdentifier(fr.Email)));
+                        var emails = fr.Email.Split(";");
```

Comment:
Create helper methods here instead. With only one line in here
participants.AddRange(SplitAndCreateBuilderParticipantsFromEmailList(fr.Email));

and same for InformationEmail
participants.AddRange(SplitAndCreateBuilderParticipantsFromEmailList(fr.InformationEmail));